src/main/java/com/binance/dex/api/client/domain/stake/Delegation.java

```diff
@@ -1,29 +1,29 @@
-package com.binance.dex.api.client.domain.stake.sidechain;
+package com.binance.dex.api.client.domain.stake;
 
 import com.binance.dex.api.client.encoding.message.Token;
 
 /**
  * @author Fitz.Lu
  **/
-public class SideChainDelegation {
```

Comment:
Is the renaming of class names and package names necessary?
We need to point this change out in release, to notify our users.

Reply:
ok, we just copy these files.